educhain/utils/output_formatter.py

```
@@ -0,0 +1,166 @@
+        question_text = Paragraph(f"Q{question.get('id', '')}: {question.get('question', '')}", 
+                                styles['Question'])
+        elements.append(question_text)
+        elements.append(Spacer(1, 12))
+
+        # Options (if present)
+        if 'options' in question:
+            options = question['options']
+            if isinstance(options, str):
+                try:
+                    options = json.loads(options)
+                except:
```

Ruff

Do not use bare `except`

(E722)